tests/cephadm/test_admin_add_labels.py

```diff
@@ -0,0 +1,78 @@
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def adding_label(instance):
+    """
+    Method add multiple labels in the node
```

Comment:
Update this to `Add multiple labels to host`

Reply:
Thanks @vamahaja
Updated